Admin/Extension/ChildExtension.php

```diff
@@ -3,12 +3,13 @@
 /*
  * This file is part of the Symfony CMF package.
  *
- * (c) 2011-2015 Symfony CMF
+ * (c) 2011-2016 Symfony CMF
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
 
+
```

Comment:
@GrahamCampbell the namespace is on the line below - it should come after the copyright header. our styleci config is quite minimal: https://github.com/symfony-cmf/core-bundle/blob/master/.styleci.yml